lib/info-extras.js

```diff
@@ -18,7 +18,10 @@ const getMetaItem = (body, name) => util.between(body, `<meta itemprop="${name}"
 exports.getVideoDescription = html => {
   const regex = /<p.*?id="eow-description".*?>(.+?)<\/p>[\n\r\s]*?<\/div>/im;
   const description = html.match(regex);
-  return description ? Entities.decode(util.stripHTML(description[1])) : '';
+  if (description) {
+    return Entities.decode(decodeURIComponent(util.stripHTML(description[1])));
```

Comment:
hmm, still getting the decode error with this video https://www.youtube.com/watch?v=PGqU_stcXDw

are you trying to fix URLs in descriptions? if so, then you can give the `.replace` calls inside `stripHTML` a function https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#Specifying_a_function_as_a_parameter

also, it would be out of this scope of this PR